common/scala/src/main/scala/whisk/core/entity/WhiskStore.scala

```diff
@@ -243,10 +243,8 @@ object WhiskEntityQueries {
         upto: Option[Instant] = None,
         convert: Option[JsObject => Try[T]])(
             implicit transid: TransactionId): Future[Either[List[JsObject], List[T]]] = {
-        implicit val ec = db.executionContext
         val startKey = List(since map { _.toEpochMilli } getOrElse 0)
         val endKey = List(upto map { _.toEpochMilli } getOrElse TOP, TOP)
-        val includeDocs = convert.isDefined
```

Comment:
This probably came up as an `unused variable` - it use to replace `convert` in the next line (but we've disabled full docs some time ago - and until streaming is added, I don't see us enabling this. So I think it's OK.